cdpctl/validation/infra/issue_templates.yml

```diff
@@ -0,0 +1,137 @@
+---
+id: AWS_ACCOUNT_ID_NOT_IN_CROSS_ACCOUNT_ROLE
+summary: Account ID {0} not in Cross-account access IAM role {1}
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-create-credentialrole.html
+---
+id: AWS_EXTERNAL_ID_NOT_IN_CROSS_ACCOUNT_ROLE
+summary: External ID {0} not in Cross-account access IAM role {1}
+docs_link: https://docs.cloudera.com/cdp/latest/requirements-aws/topics/mc-create-credentialrole.html
+---
+id: AWS_IDBROKER_INSTANCE_PROLFILE_NEEDS_ROLE
+summary: "IDBroker instance profile {0} should contain an IDBroker role."
+---
+id: AWS_IDBROKER_ROLE_NEED_EC2_TRUST_POLICY
+summary: "The IDBroker role {0} should contain a trust policy for EC2"
+---
+id: AWS_ROLE_FOR_DL_BUCKET_MISSING_ACTIONS
+summary: "The role ({0}) requires the following actions for the Datalake S3 bucket ({1})."
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_ALL_S3_RESOURCES
+summary: "The role ({0}) requires the following actions for all S3 resources"
+render_type: list
+---
+id: AWS_ROLE_FOR_DATA_BUCKET_MISSING_ACTIONS
+summary: "The role ({0}) requires the following actions for the S3 data location ({1})."
+render_type: list
+---
+id: AWS_IDBROKER_ROLE_REQUIRES_ACTIONS_FOR_ALL_RESOURCES
+summary: "The role ({0}) requires the following actions for resource wildcard (*)."
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_ALL_EC2_RESOURCES
+summary: "The role ({0}) requires the following actions for all EC2 resources ([*])"
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_ALL_RESOURCES
+summary: "The role ({0}) requires the following actions for all resources ([*])"
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_SERVICE_ROLL_RESOURCES
+summary: "The role ({0}) requires the following actions for all resources (\"arn:aws:iam::*:role/aws-service-role/*\")"
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_LOG_PATH
+summary: "The role ({0}) requires the following actions for the log storage path ({1}):"
+render_type: list
+---
+id: AWS_ROLE_REQUIRES_ACTIONS_FOR_LOG_BUCKET
+summary: "The role ({0}) requires the following actions for the log storage bucket ({1}):"
+render_type: list
+---
+id: AWS_LOGGER_INSTANCE_PROFILE_SHOULD_CONTAIN_LOGGER_ROLE
+summary: "The logger instance profile {0} set in config {1} should contain a logger role."
+---
+id: AWS_LOGGER_ROLE_SHOULD_HAVE_EC2_TRUST
+summary: "The logger role {0} should contain a trust policy for EC2"
+---
+id: AWS_S3_BUCKET_INVALID
+summary: "The s3a url {0} is invalid."
+---
+id: AWS_S3_BUCKET_DOES_NOT_EXIST
+summary: "S3 bucket {0} does not exist."
+---
+id: AWS_S3_BUCKET_FORBIDDEN_ACCESS
+summary: "S3 bucket {0} has forbidden access."
+---
+id: AWS_NON_CCM_DEFAULT_SG_NEEDS_TO_ALLOW_CDP_CIDRS
+summary: "When not using CCM (tunnel = false), the default security group {0} is missing TCP port 9443 to be allowed access from the following Cloudera CDP CIDRs"
```

Comment:
maybe rephrase to: 
The default security group {0} is missing TCP port 9443 for the following Cloudera CDP CIDRs when not using CCM (env:tunnel = false).

Reply:
done